src/WireMock.Net/Server/WireMockServer.Admin.cs

```diff
@@ -555,11 +555,64 @@ private ResponseMessage MappingsPost(RequestMessage requestMessage)
 
         private ResponseMessage MappingsDelete(RequestMessage requestMessage)
         {
-            ResetMappings();
+            // if we allow body for HTTP Delete & a body is defined
+            if (!String.IsNullOrEmpty(requestMessage.Body))
+            {
+                IEnumerable<Guid> deletedGuids = MappingsDeleteMappingFromBody(requestMessage);
```

Comment:
just use `var deletedGuids = `

Reply:
Just curious, what are the advantages of declaring ```deletedGuids``` var as opposed to IEnumerable<Guid> ?

Reply:
I normally use always var for complex objects and lists/enumerable and so on. (Except when the logic is difficult to understand.

Only for primitive types like bool, int, string.